tests/lecture.test.js

```diff
@@ -0,0 +1,84 @@
+const request = require('supertest');
+
+const { createApp } = require('../app');
+const { appDataSource } = require('../api/models/data_source');
+
+describe('logIn', () => {
```

Comment:
해당 테스트를 설명하는 문구로 수정해 주세요.

```suggestion
describe('logIn', () => {
```

Reply:
해당 테스트를 login에서 get data for reservation page로 수정하였습니다